mode/examples/Topics/Image Processing/Brightness/Brightness.pyde

```diff
@@ -29,8 +28,8 @@ def draw():
             loc = x + y * img.width
             # Get the R,G,B values from image
             r = 0
-            g = 0
-            b = 0
+            # g = 0
```

Comment:
How do you feel about a comment(s) noting that it's possible to use channels other than/in combo with `r`?